SUNRENBROXA/chains.json

```diff
@@ -6,8 +6,8 @@
       "name": "bitsong",
       "address": "bitsongvaloper1zy9xd2lz6df6gf55nq0ldase9sxtxlksdvfum2",
       "restake": {
-        "address": "bitsong18vt2a2cst3962swp2t3384lm8d7e827j5fsylp",
-        "run_time": "every 5 minutes",
+        "address": "bitsong1xllz2yrx4llfwm792geccfz7rey72m0dt6wjxg",
+        "run_time": "every day",
```

Comment:
These should all be 'every 1 day' and ideally would be a specific run time in UTC instead like '09:00'. It will be treated as midnight UTC otherwise